testsuite/features/secondary/srv_push_package.feature

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2015-2024 SUSE LLC
 # Licensed under the terms of the MIT license.
-
+# in github validation, mgr-push is not preinstalled
```

Comment:
remove comment

Reply:
review if we can't not install it but mount it from host, so it is the latest dev version

Reply:
I created a separate issue